### ISSUE TITLE
h3droid repo is empty !

### ISSUE THREAD

**Issue by pmp-p** [kotc/h3droid#1]
we may need at least some sections to store things that are not bundled in image

**drivers** for optionnal peripherals

**Apps** tweaked for h3droid both source and apk built available in folder.

**howto** in pdf formats and translated.

**Comment by sannee** [comment]
Hello. Is there any way how can i get source code of h3droid 1.3.4 ?  I need to recompile it with my own hdmi configuration in hdmi_core.c, hdmi_bsp_sun8iw7.c
I've some hdmi sound issue when i use built in 1360x768 resolution, I figured out that i need to update video_timing and ptbl arrays with my own values coresponding to my TV.  I get it fixed for armbian, but now i also want to get it fixed for h3droid. Please reply...

**Comment by pmp-p** [comment]
You'll have to use the H2 BSP

**Comment by sannee** [comment]
> You'll have to use the H2 BSP 

I built it, but video output gone after replacing h3droid kernel with H2 BSP. Unfortunately i did not figure out what exactly happened, but i was able to fix my issue with TV in another way. I cut compressed kernel (LZO format) from zImage, Then I extracted linux kernel from the LZO and patched it directly using hex editor. After that, I compressed it back and  pushed the new lzo file back to zImage. Once I replaced h3droid_zImage.u with my own patched kernel, it started work as expected this time.

Thanks,
Alex.
